etc/skel/.config/hypr/hyprland.conf

```diff
@@ -68,289 +94,369 @@ decoration {
-windowrule=pin,danmufloat
-windowrule=rounding 5,danmufloat
-windowrule=size 960 540,danmufloat
-windowrule=float,termfloat
-windowrule=move 25%-,termfloat
-windowrule=size 960 540,termfloat
-windowrule=rounding 5,termfloat
-windowrule=float,nemo
-windowrule=move 25%-,nemo
-windowrule=size 960 540,nemo
-windowrule=opacity 0.95,title:Telegram
-windowrule=opacity 0.95,title:QQ
-windowrule=opacity 0.95,title:NetEase Cloud Music Gtk4
-windowrule=animation slide right,kitty
-windowrule=animation slide right,alacritty
-windowrule=float,ncmpcpp
-windowrule=move 25%-,ncmpcpp
-windowrule=size 960 540,ncmpcpp
-windowrule=noblur,^(firefox)$
-windowrule=noblur,^(waybar)$
+windowrulev2 = opacity 0.92, class:^(thunar|nemo)$
+windowrulev2 = opacity 0.96, class:^(discord|armcord|webcord)$
+windowrulev2 = opacity 0.95, title:^(QQ|Telegram)$
+windowrulev2 = opacity 0.95, title:^(NetEase Cloud Music Gtk4)$
+# General window rules
+windowrulev2 = float, title:^(Picture-in-Picture)$
+windowrulev2 = size 960 540, title:^(Picture-in-Picture)$
+windowrulev2 = move 25%-, title:^(Picture-in-Picture)$
+windowrulev2 = float, title:^(imv|mpv|danmufloat|termfloat|nemo|ncmpcpp)$
+windowrulev2 = move 25%-, title:^(imv|mpv|danmufloat|termfloat|nemo|ncmpcpp)$
+windowrulev2 = size 960 540, title:^(imv|mpv|danmufloat|termfloat|nemo|ncmpcpp)$
+windowrulev2 = pin, title:^(danmufloat)$
+windowrulev2 = rounding 5, title:^(danmufloat|termfloat)$
+windowrulev2 = animation slide right, class:^(kitty|Alacritty)$
+windowrulev2 = noblur, class:^(org.mozilla.firefox)$
+# Decorations related to floating windows on workspaces 1 to 10
+windowrulev2 = bordersize 2, floating:1, onworkspace:w[fv1-10]
+windowrulev2 = bordercolor $cachylblue, floating:1, onworkspace:w[fv1-10]
+windowrulev2 = rounding 8, floating:1, onworkspace:w[fv1-10]
+# Decorations related to tiling windows on workspaces 1 to 10
+windowrulev2 = bordersize 3, floating:0, onworkspace:f[1-10]
+windowrulev2 = rounding 4, floating:0, onworkspace:f[1-10]
+# Windows Rules End #
+
+# Workspaces Rules https://wiki.hyprland.org/0.45.0/Configuring/Workspace-Rules/ #
+# workspace = 1, default:true, monitor:$priMon
+# workspace = 6, default:true, monitor:$secMon
+# Workspace selectors https://wiki.hyprland.org/0.45.0/Configuring/Workspace-Rules/#workspace-selectors
+# workspace = r[1-5], monitor:$priMon
+# workspace = r[6-10], monitor:$secMon
+# workspace = special:scratchpad, on-created-empty:$applauncher
+# no_gaps_when_only deprecated instead workspaces rules with selectors can do the same
+# Smart gaps from 0.45.0 https://wiki.hyprland.org/0.45.0/Configuring/Workspace-Rules/#smart-gaps
+workspace = w[tv1-10], gapsout:5, gapsin:3
+workspace = f[1], gapsout:5, gapsin:3
+# Workspaces Rules End #
+
+# Layers Rules #
+layerrule = animation slide top, logout_dialog
+# layerrule = animation popin 50%, waybar
+layerrule = animation slide down, waybar
+layerrule = animation fade 50%, wallpaper
+layerrule = noblur, waybar
```

Comment:
This one causes an error because its "invalid" due to the rule not existing

Reply:
Oh, sorry! Strange, really shouldn't work because it doesn't exist (I don't why I put it there), but did not give an error.
Removing it

Reply:
I did not see that error because that wasnt on hyprland.conf was on a copy that I had to make because HyprLS/Hyprlang wasn't in any way code coloring my hyprland.conf. So I created a copy dhyrprland.conf, since on any other name .conf it works.

sorry about the delays, energy problems on my city.

Reply:
No worries, they're some conflicts with the file which have to be resolved in order to approve & merge

Reply:
I think I updated everything.